test/unittesting/testingutils.go

```diff
@@ -424,11 +397,7 @@ func ExtractInfoFromResponseWhenAntiCSRFisNone(res *http.Response) map[string]st
 
 func getInstallationDir() string {
 	installationDir := os.Getenv("INSTALL_DIR")
-	if installationDir == "" {
-		installationDir = returnNumberOfDirsToGoUpFromCurrentWorkingDir(false) + "supertokens-root"
-	} else {
-		installationDir = returnNumberOfDirsToGoUpFromCurrentWorkingDir(true) + installationDir
-	}
+	installationDir = "../../" + installationDir
```

Comment:
This change will require a change in the contributing docs too right?

Reply:
Ya It will

Reply:
So please make them in this PR